validator/client/runner.go

```diff
@@ -75,7 +75,7 @@ func run(ctx context.Context, v iface.Validator) {
 		if err != nil {
 			log.Fatalf("Could not determine if beacon node synced: %v", err)
 		}
-		err = v.WaitForActivation(ctx)
+		err = v.WaitForActivation(ctx, nil)
```

Comment:
This is a little weird without context, I would recommend a comment like
`nil /* something something */`